frontend/src/features/boards/components/TaskList.vue

```diff
@@ -0,0 +1,199 @@
+
+import { state } from "@/modules/store";
+import { Board, Column } from "@f/boards/types";
+import { formatTimeSince } from "@f/utils/date";
+
+const input = ref();
+const route = useRoute();
+const newTasklistOpen = ref(false);
+const newTaskContent = ref("");
+
+const list = defineModel<Column>({ required: true });
+const board = computed<Board>(() => state.boards[route.params.id.toString()]);
+const since = computed<string>(() => formatTimeSince(new Date(list.value.createdAt || Date.now())));
+
+const addTaskList = (): void => {
+  const id: string = uid();
+  list.value.tasks[id] = {
+    id: id,
+    order: Object.keys(list.value.tasks).length + 1,
+    content: newTaskContent.value,
+  };
+  newTaskContent.value = "";
+  input.value.focus();
+};
+
+const deleteTaskList = (): void => {
+  delete board.value.columns[list.value.id];
+};
+</script>
+
+<template>
+  <q-card
+    class="tasklist"
+    :style="`backgroundColor: ${list.color || ''}`"
+  >
+    <q-card-section class="q-pa-sm">
+      <div class="row items-center justify-between">
+        <div class="text-subtitle1 auto-invert">
+          {{ list.name }}
+        </div>
+
+        <q-btn
+          round
+          size="sm"
+          flat
+        >
+          <q-icon name="settings" />
+        </q-btn>
+        <q-popup-proxy>
+          <q-card class="q-pa-sm popup">
+            <q-card-section class="no-padding">
+              <q-item>
+                <q-avatar
+                  color="primary"
+                  size="xl"
+                />
+                <div class="column q-pl-md">
+                  <b>Owner</b>
+                  <div>{{ since }}</div>
+                </div>
+              </q-item>
+              <q-item
+                clickable
+                :style="`backgroundColor: ${list.color || ''}`"
+              >
+                <q-item-section avatar>
+                  <q-icon
+                    color="primary"
+                    name="palette"
+                  />
+                </q-item-section>
+
+                <q-item-section :style="`backgroundColor: ${list.color || ''}`">
+                  <q-item-label />
+
+                  <q-item-label
+                    class="auto-invert"
+                    caption
+                  >
+                    {{ list.color || `Background Color` }}
+                  </q-item-label>
+                </q-item-section>
+
+                <q-popup-proxy>
+                  <q-color v-model="list.color" />
+                </q-popup-proxy>
+              </q-item>
+
+              <q-item
+                clickable
+                @click="deleteTaskList"
+              >
+                <q-item-section avatar>
+                  <q-icon
+                    color="red"
+                    name="delete"
+                  />
+                </q-item-section>
+
+                <q-item-section>
+                  <q-item-label class="text-bold text-red"> DELETE CARD </q-item-label>
```

Comment:
```suggestion
                  <q-item-label class="text-bold text-red text-uppercase">Delete card</q-item-label>
```
Best practice is to not hard-code uppercase text. Use normal text and capitalise via CSS. This makes it easy to change via styles and when the user copies it, it is in standard case.